roles/postgres/templates/postgres.yaml.j2

```diff
@@ -43,6 +43,24 @@ spec:
         - image: '{{ postgres_image }}'
           name: postgres
           env:
+            # For tower_postgres_image based on rhel8/postgresql-12
```

Comment:
shouldn't it be awx_postgres_image?

Reply:
well really hub base image. I'll fix the comment in my pr.